app/project/[id]/components/project-files.tsx

```diff
@@ -21,7 +27,29 @@ import { Input } from "@/components/ui/input";
 
 import { DesignFile, DesignProject, DesignTask } from "@/lib/types";
 import DateTime from "@/components/date-time";
-import { cn } from "@/lib/utils";
+import DeleteAlertDialog from "@/components/delete-alert";
+import { deleteFile } from "@/lib/actions";
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { DataTablePagination } from "@/components/ui/pagination";
+
+const deleteProfileAvatar = (id: number) => {
+  try {
+    deleteFile(id);
+    location.reload();
+  } catch (error) {
+    console.log(`Error happened:`, error);
+  }
+};
+
+const columnLabels: { [key: string]: string } = {
+  id: "Delete",
```

Comment:
id?? seems incorrect
does the data table _need_ `accessorKey`?

Reply:
Need the id to pass to the Delete function. Is there any way? if there please mention this

Reply:
I did some looking up of the tanstack documentation and it seems there are multiple column types but the objects we use here default to the "accessor" column type
https://tanstack.com/table/v8/docs/guide/column-defs#column-def-types

we can use the column helper to specify we want a "display" type header instead `const columnHelper = createColumnHelper<Person>()` and
```
columnHelper.display({
    id: "delete",
    header: () => <div ...
```

Reply:
actually, we might not even need the column helper, if we change the accessorKey to id, I think the type is infered to be a display column rather than a accessor column!